airflow/contrib/auth/backends/github_enterprise_auth.py

```diff
@@ -154,9 +154,10 @@ def ghe_team_check(self, username, ghe_token):
                     resp.status if resp else 'None'))
 
         for team in resp.data:
-            # team json object has a slug cased team name field aptly named
-            # 'slug'
-            if team['slug'] in teams:
+            # mylons: teams are part of organizations. organizations are unique, but teams are not
+            # therefore 'slug' for a team is not necessarily unique.
+            # use id instead
```

Comment:
this comment doesn't really make sense without the context that the following line used to be `team['slug']`